docs/config/README.md

```diff
@@ -2276,17 +2276,24 @@ Multiple custom modules can be defined by using a `.`.
 
 ::: tip
 
-The order in which custom modules are shown can be individually set
-by setting `custom.foo` in `prompt_order`. By default, the `custom` module
-will simply show all custom modules in the order they were defined.
+The order in which custom modules are shown can be individually set by including 
+`${custom.foo}` in `prompt_order` (as it includes a dot, you need to use `${...}`). 
```

Comment:
I think this should actually be since for the format string release we got rid of the prompt_order and replaced it with a top level format string.

```suggestion
`${custom.foo}` in the top level `format` (as it includes a dot, you need to use `${...}`). 
```

Reply:
Also removed prompt_order in the FAQ and in some debug prints